docs/pages/text.js

```diff
@@ -10,23 +10,164 @@ import MainSection from '../components/MainSection.js';
+</Flex>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Start-align paragraph text."
+            defaultCode={`
+<Text>Our mission is to bring everyone the inspiration to create a life they love. To do that, we show you personalized content and ads we think you’ll be interested in based on information we collect from you and third parties. We only use that information where we have a proper legal basis for doing so.</Text>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Center-align paragraph text. This is hard to read, especially for users with dyslexia."
+            defaultCode={`
+<Text align="center">Our mission is to bring everyone the inspiration to create a life they love. To do that, we show you personalized content and ads we think you’ll be interested in based on information we collect from you and third parties. We only use that information where we have a proper legal basis for doing so.</Text>
+`}
+          />
+        </MainSection.Subsection>
+        <SlimBanner
+          type="info"
+          iconAccessibilityLabel="Info"
+          message="Gestalt's typography guidelines contain additional best practices around sizing, style and hierarchy."
+          helperLink={{
+            text: 'View Typography guidelines',
+            accessibilityLabel: 'View Typography guidelines',
+            href: '/typography',
+            onClick: () => {},
+          }}
+        />
+      </MainSection>
 
+      <MainSection name="Accessibility">
+        <MainSection.Subsection
+          title="Accessible sizing"
+          description="A minimum text size of 16 px (12pt) is recommended for readability. Some short text labels, or secondary text can go lower than that, but smaller sizes should be kept to a minimum. Making text brief will also help with readability."
+        />
+        <MainSection.Subsection
+          title="Accessible color"
+          description="For low-vision users, text color contrast is very important. To insure accessible contrast, stick to our [standard text colors](/color_usage#Standard-text-colors). See our [accessibility](/accessibility) page for design considerations and handy accessibility tools for checking color contrast."
+        />
+      </MainSection>
+      <MainSection
+        name="Localization"
+        description="Keep text simple and short to avoid truncation or line wrapping in UI controls like buttons when translating languages that require more characters. Avoid overriding our line-height settings, as this can result in text clipping for scripts, like Hindi, that have taller ascenders and descenders."
+      >
+        <MainSection.Subsection
+          title="Text-wrapping and hyphenation"
```

Comment:
We don't mention iOS in any other cmp page. Should we remove this?

Reply:
Honestly, I have no context for this addition. I think this is definitely something we should evaluate in a future pass.